### PR TITLE
Import mega-linter.yml

### LINKED ISSUE #147: Follow and consider if mega-linter could be a working/safe alternative to super-linter
https://nvuillam.github.io/mega-linter/
https://nvuillam.github.io/mega-linter/#mega-linter-vs-super-linter

> Mega-Linter can automatically apply fixes performed by linters, and push them to the same branch, or create a Pull Request that you can validate
> This is pretty handy, especially for linter errors related to formatting (in that case, you don’t have any manual update to perform)

### PR DESCRIPTION
**Pull request by gwarf**
Import mega-linter.yml, to test [mega-linter](https://megalinter.io/latest/).
Include file mentioend at https://megalinter.io/latest/install-github/, but at least currently, do not commit directly to the PR, but create a new PR, to be tested/reviewed.

Fix #147.

### CODE REVIEW

**Review by enolfc — COMMENTED** [review]
mmm, the linter code does not pass linting 🤔

**Comment by gwarf** [comment]
Looking at configuring megalinter, I tend to like a lot how it's easy to identify how to configure or disable linters, there is a direct link in the test output to the megalinter documentation for a given linter, providing information on the linter home page, it's configuration and so on, such as: https://megalinter.io/v6/descriptors/copypaste_jscpd/

It's very convenient.

**Comment by gwarf** [comment]
It finally works :tada:  :)

File: .github/workflows/mega-linter.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            github.event_name) && env.APPLY_FIXES_MODE == 'commit' && github.ref
+            != 'refs/heads/main' && (github.event_name == 'push' ||
+            github.event.pull_request.head.repo.full_name == github.repository) &&
+            !contains(github.event.head_commit.message, 'skip fix')
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          branch: ${{ github.event.pull_request.head.ref || github.head_ref || github.ref }}
+          commit_message: "[MegaLinter] Apply linters fixes"
+          commit_user_name: megalinter-bot
+          commit_user_email: nicolas.vuillamy@ox.security
```
Comment (enolfc) [inline .github/workflows/mega-linter.yml:122]: Should we use this email here?

**Review by enolfc — COMMENTED** [review]
What will happen to the super-linter? do we keep it also?

**Comment by gwarf** [comment]
> What will happen to the super-linter? do we keep it also?

I was expecting to still keep it for some time, till we have more experience and can drop one of the two

File: .github/workflows/mega-linter.yml (hunk 1 above)
Comment (gwarf) [reply .github/workflows/mega-linter.yml:122]: I guess not. Maybe we could reuse/revive https://github.com/egibot for this ?

Comment (gwarf) [reply .github/workflows/mega-linter.yml]: I've pushed a commit to address this, please have a look.

**Review by enolfc — APPROVED** [review]
LGTM, let's try it

**Pull request merged**